CHANGELOG.md

```diff
@@ -8,17 +8,27 @@ accidentally triggering the load of a previous DB version.**
 
 **Minor features**
 * #2736 Support adding columns to hypertables with compression enabled
+
+## 2.0.2 (2021-02-19)
+
```

Comment:
Missing some textual description of the release:
Something like
```
This maintenance release contains bugfixes since the 2.0.1 release.
We deem it high priority for upgrading.

In particular the fixes contained in this maintenance release address
issues in continuous aggregates, compression, JOINs with hypertables
and background jobs.
```

Reply:
Added to commit message but forgot to add here.